arwen/host/execution_async.go

```diff
@@ -98,12 +100,19 @@ func (host *vmHost) postprocessCrossShardCallback() error {
 		return err
 	}
 
+	// If the asyncContext is empty, but execution has reached here, it means
+	// that we're executing a legacy callback, which should be reconstructed.
+	// TODO this might be completely unnecessary
+	if !asyncContext.HasPendingCallGroups() {
```

Comment:
we just need to simply execute the "callBack" function

Reply:
Deleted.